scripts/package-emulator.sh

```diff
@@ -1,33 +1,24 @@
 #!/bin/bash
-set -
-cd ..
-. setup.sh
+source build/envsetup.sh
+HOST_ARCH=$(get_build_var HOST_ARCH)
+lunch "aosp_${HOST_ARCH}-userdebug"
```

Comment:
The step `. setup.sh` will lunch the right build target. Is there a special reason need to lunch host arch and userdebug? I think this will cause a problem, when we build arm simulator, the emulator arm is different to the host.

Reply:
No particular reason, this was just quick hacking and something wasn't working, I'll just revert this.